applications/template/cookiecutter.json

```diff
@@ -0,0 +1,14 @@
+{
+    "project_name": "My Project",
+    "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
+    "full_name": "John Doe",
+    "affiliation": "My Institution",
```

Comment:
Could we add [human readable prompts](https://cookiecutter.readthedocs.io/en/stable/advanced/human_readable_prompts.html) for this information?